plugin-gradle/CHANGES.md

```diff
@@ -3,6 +3,7 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `3.27.0`).
 
 ## [Unreleased]
+* Accept `java.nio.charset.Charset` type when setting the character encoding via `encoding` ([#1128](https://github.com/diffplug/spotless/issues/1128))
```

Comment:
```
## [Unreleased]
### Added
* Accept ...
```

Reply:
Ah! Didn't see this one in the previous list!

I'll leave the requests "unresolved" because I think you would like to actually resolve them once you see that they are fixed or not relevant anymore :grin: